unitTests/ui/CategoryFacet/CategoryFacetSearchTest.ts

```diff
@@ -11,14 +11,29 @@ import { analyticsActionCauseList } from '../../../src/ui/Analytics/AnalyticsAct
 import * as Globalize from 'globalize';
 import { Simulate } from '../../Simulate';
 
+function undebounce(fn: (...args: any[]) => any, time = 0): (...args: any[]) => void {
+  return (...args) => {
+    let wasClockInstalled = false;
+    try {
+      jasmine.clock().install();
+    } catch (e) {
+      wasClockInstalled = true;
+    }
+    fn(...args);
+    jasmine.clock().tick(time);
+    if (!wasClockInstalled) {
+      jasmine.clock().uninstall();
```

Comment:
In the red-diff below, we spied on the debounce returned a function that called the passed function (similar to here).
However, we were not using clock before. Why is it necessary now?

Reply:
Before, the debounce function was replaced to run everything synchronously. Here, since underscore no longer allows us to overwrite its function, I used a clock to skip the debounce time.